portfolio/src/components/Home.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Navbar from '../layouts/Navbar';
+import home from '../44980855102_a4c36d142a_o.jpg'
+
+class Home extends Component {
+    render() {
+        return (
+            <body>
+            <Navbar />
+        <div className="content">
+            <div className="subcontent">
+                <div className="cont">
+                <span style={{color:"#F3C00A"}}>Hello!.</span>
+            I`m <span style={{color:"#F3C00A"}}>Nengo Ally</span>
+            a <span style={{color:"#F3C00A"}}>software developer</span>
+            and <span style={{color:"#F3C00A"}}>public-speaker</span> based
```

Comment:
@Ally4 Instead of duplicating the same color code, you can create one class for them.